### DISCUSSION TITLE
salt-key --finger-all or just --finger don't work with listing

### DISCUSSION THREAD

**Discussion by poige** [saltstack/salt#67618] [Feature Requests]
Salt: 2019.2.0

Say, if a user has some keys denied they can be listed with just:

`salt-key -l denied`

But it's impossible to get these keys fingerprints displayed adding `-F` option.

**Comment by poige** [comment]
It probably makes sense to elaborate why this could be needed and such issues are already filled in, for e. g.: #46045

**Comment by frogunder** [comment]
@poige Thank you for reporting this issue.

Will label as a feature request.

@saltstack/team-core Any additional thoughts on this?
